{"issue_url": "https://api.github.com/repos/rails/rails/issues/3329", "repo": "rails", "body": "@aka47 I'm also hitting this issue if a newly added record has the same values on the unique index as a record marked for destruction. Do you have a fix for this, or did you simply just remove the unique index? I'd rather not remove the unique index.", "_id": {"$oid": "523837a7bd3543c15100fea5"}, "issue_id": 3329, "url": "https://api.github.com/repos/rails/rails/issues/comments/15859814", "html_url": "https://github.com/rails/rails/pull/3329#issuecomment-15859814", "updated_at": "2013-04-03T19:43:54Z", "user": {"subscriptions_url": "https://api.github.com/users/whather/subscriptions", "events_url": "https://api.github.com/users/whather/events{/privacy}", "repos_url": "https://api.github.com/users/whather/repos", "gists_url": "https://api.github.com/users/whather/gists{/gist_id}", "url": "https://api.github.com/users/whather", "login": "whather", "gravatar_id": "abd4fa28791874b2c95655864436ebee", "html_url": "https://github.com/whather", "following_url": "https://api.github.com/users/whather/following{/other_user}", "received_events_url": "https://api.github.com/users/whather/received_events", "organizations_url": "https://api.github.com/users/whather/orgs", "avatar_url": "https://0.gravatar.com/avatar/abd4fa28791874b2c95655864436ebee?d=https%3A%2F%2Fidenticons.github.com%2Fd0f091c27fca89c0cc91fa07916feb7f.png", "starred_url": "https://api.github.com/users/whather/starred{/owner}{/repo}", "id": 148534, "followers_url": "https://api.github.com/users/whather/followers", "type": "User"}, "created_at": "2013-04-03T19:43:54Z", "id": 15859814, "owner": "rails"}
